csharp/Platform.IO/FileHelpers.cs

```diff
@@ -86,12 +242,48 @@ public static void SetSize(string path, long size)
             }
         }
 
+        /// <summary>
+        /// <para>Removes all files from the directory at the path <paramref name="directory"/>.</para>
+        /// <para>Удаляет все файлы из директории находящейся по пути <paramref name="directory"/>.</para>
+        /// </summary>
+        /// <param name="directory">
+        /// <para>The path to the directory to be cleaned.</para>
+        /// <para>Путь к директории для очистки.</para>
+        /// </param>
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
         public static void DeleteAll(string directory) => DeleteAll(directory, "*");
 
+        /// <summary>
+        /// <para>Removes all files from the directory at the path <paramref name="directory"/> according to the <paramref name="searchPattern"/>.</para>
+        /// <para>Удаляет все файлы из директории находящейся по пути <paramref name="directory"/> в соотвествии с шаблоном поиска <paramref name="searchPattern"/>.</para>
+        /// </summary>
+        /// <param name="directory">
+        /// <para>The path to the directory to be cleaned.</para>
+        /// <para>Путь к директории для очистки.</para>
+        /// </param>
+        /// <param name="searchPattern">
+        /// <para>Search pattern for files in the directory at the path <paramref name="directory"/>.</para>
+        /// <para>Шаблон поиска для файлов в директории находящейся по пути <paramref name="directory"/>.</para>
+        /// </param>
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
         public static void DeleteAll(string directory, string searchPattern) => DeleteAll(directory, searchPattern, SearchOption.TopDirectoryOnly);
 
+        /// <summary>
+        /// <para>Removes all files from the directory at the path <paramref name="directory"/> according to the <paramref name="searchPattern"/> and the <paramref name="searchOption"/>.</para>
+        /// <para>Удаляет все файлы из директории находящейся по пути <paramref name="directory"/> в соотвествии с шаблоном поиска <paramref name="searchPattern"/> и настройкой поиска <paramref name="searchOption"/>.</para>
+        /// </summary>
+        /// <param name="directory">
+        /// <para>The path to the directory to be cleaned.</para>
+        /// <para>Путь к директории для очистки.</para>
+        /// </param>
+        /// <param name="searchPattern">
+        /// <para>Search pattern for files in the directory at the path <paramref name="directory"/>.</para>
+        /// <para>Шаблон поиска для файлов в директории находящейся по пути <paramref name="directory"/>.</para>
+        /// </param>
+        /// <param name="searchOption">
+        /// <para>Specifies whether to search only in the current the directory at the path <paramref name="directory"/>, or also in subdirectories.</para>
```

Comment:
`also in subdirectories` => `also in all subdirectories`

Reply:
`Specifies whether` => `A <see cref="SearchOption"/> value that determines whether`